.travis.yml

```diff
@@ -1,9 +1,15 @@
 language: python
+sudo: false
+cache:
+  directories:
+    - node_modules
```

Comment:
the bower install is very fast. It doesn't needs 3 seconds. For that reason the cache would slow down travis.

See http://docs.travis-ci.com/user/caching/#Things-not-to-cache